Makefile

```diff
@@ -18,27 +18,40 @@ CPPCHECK=ls lib/*.cc khmer/_khmermodule.cc | grep -v test | cppcheck -DNDEBUG \
 	 --quiet -Ilib -Ithird-party/bzip2 -Ithird-party/zlib \
 	 -Ithird-party/smhasher
 
-all: khmer/_khmermodule.so
 
+## all                  : default task; show list of build commands
+all: commands
+
+commands: Makefile
+	@sed -n 's/^##//p' $<
+
+## install-dependencies : use pip to install Python module dependencies
 install-dependencies:
 	pip2 install --upgrade $(DEVPKGS) || pip install --upgrade $(DEVPKGS)
 
+## sharedobj            : build khmer shared object file
+sharedobj: khmer/_khmermodule.so
+
 khmer/_khmermodule.so: $(CPPSOURCES)
 	./setup.py build_ext --inplace
 
+## coverage-debug       : debug code test coverage
```

Comment:
I never run this target directly; do you?

Lets only document the major targets